packages/ui-components/package.json

```diff
@@ -51,7 +51,7 @@
     "@babel/core": "^7.12.13",
     "@babel/plugin-syntax-jsx": "^7.12.1",
     "@material-ui/core": "4.11.3",
-    "@typescript-eslint/eslint-plugin": "^5.15.0",
+    "@typescript-eslint/eslint-plugin": "^5.27.0",
```

Comment:
Does this package need to depend on `@typescript-eslint`?

Reply:
Good question. Yes for `react-app` this extends on line 35. https://github.com/facebook/create-react-app/blob/main/packages/eslint-config-react-app/index.js#L45